### PR TITLE
Feature/new firmware tenth of degree

### PR DESCRIPTION
**Pull request by martiniil**

### CODE REVIEW

File: src/laserscan.cpp [hunk 1]
```diff
@@ -24,15 +24,17 @@ namespace psen_scan_v2
...
   : resolution_(resolution), min_scan_angle_(min_scan_angle), max_scan_angle_(max_scan_angle)
 {
-  if (getScanResolution() == 0.)
+  if (getScanResolution() == TenthOfDegree(0))
   {
     throw std::invalid_argument("Resolution must not be 0");
   }
 
-  if (getScanResolution() < MIN_X_AXIS_ROTATION || getScanResolution() > MAX_X_AXIS_ROTATION)
+  if (getScanResolution().toRad() < MIN_X_AXIS_ROTATION || getScanResolution().toRad() > MAX_X_AXIS_ROTATION)
```
Comment (agutenkunst) [inline src/laserscan.cpp:37]: TODO

Comment (martiniil) [reply src/laserscan.cpp]: Done

File: test/unit_tests/unittest_tenth_of_degree.cpp [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+TEST(TenthOfDegreeTest, valueTest)
+{
+  TenthOfDegree tenth_of_degree{ 1 };
+  EXPECT_EQ(tenth_of_degree.value(), 1);
+}
+
+TEST(TenthOfDegreeTest, toRad)
+{
+  TenthOfDegree tenth_of_degree{ 1 };
+  EXPECT_EQ(tenth_of_degree.toRad(), tenthDegreeToRad(1));
```
Comment (martiniil) [inline test/unit_tests/unittest_tenth_of_degree.cpp:39]:
```suggestion
  EXPECT_DOUBLE_EQ(tenth_of_degree.toRad(), tenthDegreeToRad(1));
```
or did I miss something?

**Review by martiniil — COMMENTED** [review]
@agutenkunst Only one comment left from my side.

**Pull request merged**